mkdocs.yml

```diff
@@ -81,36 +81,37 @@ nav:
   - understanding_the_basics*.md
   - quickstart*.md
   - Core Concepts:
-    - core-concepts/core-concepts*.md
-    - core-concepts/config*.md
-    - core-concepts/stateconfig*.md
-    - core-concepts/variable*.md
-    - core-concepts/dependencies*.md
-    - core-concepts/output*.md
+      - core-concepts/core-concepts*.md
+      - core-concepts/config*.md
+      - core-concepts/stateconfig*.md
+      - core-concepts/variable*.md
+      - core-concepts/dependencies*.md
+      - core-concepts/output*.md
   - Commands:
-    - commands/autocomplete*.md
-    - commands/*.md
+      - commands/install*.md
+      - commands/autocomplete*.md
+      - commands/*.md
   - Plugins:
-    - plugins/plugins-list*.md
-    - plugins/*.md
+      - plugins/plugins-list*.md
+      - plugins/*.md
   - Best Practices: best-practices/
   - contributing_guide*.md
   - contributor_ladder*.md
   - Governance ⧉: https://github.com/devstream-io/devstream/blob/main/GOVERNANCE.md
   - Developer Guide:
-    - development/architecture*.md
-    - development/development-workflow*.md
-    - development/commit-messages*.md
-    - development/lint*.md
-    - development/project-layout*.md
-    - development/build*.md
-    - development/test*.md
-    - development/mkdocs*.md
-    - development/translation*.md
-    - development/creating-a-plugin*.md
-    - development/commands/develop*.md
-    - development/reviewing*.md
-    - development/good-first-issues*.md
-    - development/help-wanted*.md
-    - development/branch-and-release*.md
+      - development/architecture*.md
```

Comment:
unexpected indent

Reply:
I noticed this yesterday as well and looked through several sites and found that they all have indentation after the bullet list (like this pr). So I thought this pr was correct about the indent change.

But today I checked again and found the official site of yaml, which is not indented, so I guess we really don't need indentation. @zhenyuanlau 

https://yaml.org/

<img width="741" alt="image" src="https://user-images.githubusercontent.com/36830265/183546942-3ccfa8fd-e838-4fe1-a9fe-3fba4760de52.png">

Reply:
Very detailed work, give you a thumbs up!

Reply:
See also [[YAML] indentation style for sequence in mapping](https://github.com/prettier/prettier/issues/4723).